src/studio-header/HeaderBody.jsx

```diff
@@ -51,7 +52,7 @@ const HeaderBody = ({
   );
 
   return (
-    <Container size="xl" className="px-2.5">
+    <Container size={full ? undefined : 'xl'} className="px-2.5">
```

Comment:
If we [add a `size="full"` option](https://github.com/openedx/paragon/pull/3216#discussion_r1746562375) instead, then this becomes a lot clearer:

```suggestion
    <Container size={full ? 'full' : 'xl'} className="px-2.5">
```

Reply:
Check my comment in the Paragon PR